draft-ietf-tls-tls13.md

```diff
@@ -2925,6 +2944,7 @@ The server MUST NOT send a "psk_key_exchange_modes" extension.
 
 %%% Key Exchange Messages
 
+       /* Managed by IANA */
```

Comment:
I'm inclined not to have this. Why do people need to know this at this point?

Reply:
This one I agree is not needed.  I was first looking at the CertificateType enum, which we are not adding any values to, so is not otherwise mentioned in the IANA considerations and it seemed worth mentioning that IANA is involved in those values.  Here, we are creating a new registry, and that's enough mention of IANA.